Xamarin.Forms.Platform.iOS/VisualElementPackager.cs

```diff
@@ -20,13 +20,15 @@ public VisualElementPackager(IVisualElementRenderer renderer) : this(renderer, n
 		{
 		}
 
-		VisualElementPackager(IVisualElementRenderer renderer, VisualElement element)
+		VisualElementPackager(IVisualElementRenderer renderer, VisualElement element, bool isHeadless = false)
 		{
 			if (renderer == null)
 				throw new ArgumentNullException(nameof(renderer));
 
 			Renderer = renderer;
-			renderer.ElementChanged += OnRendererElementChanged;
+			if (!isHeadless)
```

Comment:
another option would be to keep a ref to the EventHandler, I'm quite sure you can't add it multiple times